setup.cfg

```
@@ -0,0 +1,5 @@
+[flake8]
+ignore =
```

I know this goes against the suggestion in the issue mentioned in the description, but for this PR I think it's easiest to just focus on the errors `F401,F403` (which the CI workflow is supposed to handle).

Try adding this @ilnarkz:
```suggestion
ignore =
select = F401,F403
per-file-ignores =
    cleanlab/__init__.py: F401
    cleanlab/token_classification/__init__.py: F401
    cleanlab/benchmarking/__init__.py: F401
```

---

The .pylintrc config had the setting:
https://github.com/cleanlab/cleanlab/blob/886e6aee6e81df463afbf9ff6e55a7d489343a39/.pylintrc#L408-L409

which doesn't exactly have a flake8-counterpart.

---

@elisno But then flake8 will only track these errors. Is this correct?

---

You're right. That's the only thing at the moment that pylint is doing in our CI.

To keep the scope of this PR manageable, you should aim to replace that functionality without adding anything extra.

We can always open a new PR later for tracking more (or all) errors with flake8.